doc/next/_sources/campaign-overview.rst.txt

```diff
@@ -0,0 +1,86 @@
+.. _campaign_overview:
+
+Overview
+========
+
+Campaigns allow you, the seller, to group and manage your ads together, having shared budgets, targeting rules, and other settings. You can create separate campaigns to run ads in different locations or using different budgets.
+
+Campaigns are a simpler way to organize and manage your products, services, and ads in general.
+They are becoming first-class citizen in Admarkt, and as such, are a primary container to get your advertisements online.
+
+Even though they are restricted in their current capabilities, we plan to evolve the concepts in line with other advertising platforms like Google and Facebook. We will keep you posted as we release more campaign-centric features in the future.
+
+..  include:: /campaign-restrictions.rst
+
+
+To give you an idea, it is possible to pause a campaign, resulting in all ads in the campaign to be taken offline. When a campaign reaches one of its :ref:`campaign_budgets`, all ads in that campaign will be taken offline. When budget becomes available again, all ads in the campaign will come
+online again.
+
+.. _components:
+
+Components
+----------
+
+A campaign consists of several components:
+
+========================= ================================================
+Component                 Short description
+========================= ================================================
+:ref:`campaign_id`        The unique reference to the campaign
+:ref:`campaign_title`     User-provided title
+:ref:`campaign_vendorid`  Unique reference for the seller to the campaign
+:ref:`campaign_status`    Status of the campaign
+:ref:`campaign_budgets`   Budgets of this campaign
+:ref:`campaign_targeting` Targeting options of this campaign
+========================= ================================================
+
+
+.. _campaign_id:
+
+Campaign ID
+"""""""""""
+Unique reference to the Campaign, needs to be omitted in a POST action. Generated by us,
+and needed for certain GET/PUT actions to modify the campaign and its properties.
+
+
+.. _campaign_title:
+
+Campaign title
+""""""""""""""
+The title of the campaign, provided by you. This makes it easier for you to distingish between your campaigns. Your campaign titles are not visible to your ad viewers.
```

Comment:
We currently also don't show this in the console, should we mention that? There is no way to really trace this other than API calls. However this might be different in the future, especially retail.

Reply:
Hm, good point, but this is why it says `This makes it easier for you to distingish between your campaigns.` Shall I add that the Consoel doesn't show it currently?